app/models/event.rb

```diff
@@ -193,6 +193,14 @@ def compute_occupied_places
     application_letters.where(status: ApplicationLetter.statuses[:accepted]).count
   end
 
+  # Returns the number of currently rejected applications with no mail sent of the event
+  #
+  # @param none
+  # @return [Int] for number of rejected applications
+  def compute_number_of_rejections_without_mail
```

Comment:
not wrong, but I think what we want to check is if there are rejected participants at all.
maybe we should rename the function to has_rejected_participants and perform the > 0 check in here?